dbt_subprojects/dex/models/_projects/zeroex/arbitrum/zeroex_arbitrum_settler_addresses.sql

```diff
@@ -2,9 +2,15 @@
     config(
         schema = 'zeroex',
         alias = 'arbitrum_settler_addresses',
-        materialized = 'table'
+        materialized='incremental',
+        partition_by = ['begin_block_time'],
+        unique_key = ['token_id', 'settler_address', 'begin_block_number', 'begin_block_time'],
```

Comment:
a bit confused on the unique keys. we would only update `tx_hash` then? we want these four fields to remain static? or do you want all five fields to be unique, that way it acts as append only?